models/demos/ttnn_resnet/tests/ttnn_resnet_test_infra.py

```diff
@@ -0,0 +1,259 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from loguru import logger
+import torch
+import torchvision
+
+import ttnn
+from ttnn.model_preprocessing import (
+    preprocess_model_parameters,
+)
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from models.demos.ttnn_resnet.tt.custom_preprocessing import create_custom_mesh_preprocessor
+from models.demos.ttnn_resnet.tt.ttnn_functional_resnet50_new_conv_api import resnet50
+
+
+## copied from ttlib version test:
+# golden pcc is ordered fidelity, weight dtype, activation dtype
+golden_pcc = {
+    8: {
+        (
+            ttnn.MathFidelity.HiFi4,
+            ttnn.bfloat16,
+            ttnn.bfloat16,
+        ): 0.983301,  # PCC: 0.9833017469734239             TODO: NEED DEBUGGING WHY THIS IS SLIGHTLY LOWER THAN TTLIB
```

Comment:
🐵

Reply:
This is just moved out of a test file to a new one. There was no change to these values